tests/integration/test_agentd/test_agentd_enrollment_params.py

```diff
@@ -86,6 +95,17 @@ def clean_log_file():
 
 
 def override_wazuh_conf(configuration):
+    """
```

Comment:
Add a first line that explains what it does in a single line. If you need more detail, split it and add a new paragraph.

Reply:
https://github.com/wazuh/wazuh-qa/pull/1235/commits/129ec4dd555dc61dc83723849962dc8dd861ecbb